### ISSUE TITLE
TypeError: Cannot read property 'workerPoolId' of undefined

### ISSUE THREAD

**Issue by danielfischeroctopus** [OctopusDeploy/Issues#4744]
(Tested on Octopus version `2018.7.1`)

TypeError when viewing some Step Templates.

Steps to replicate:

1. Library -> Step Templates
2. Installed Community Stem Templates `Browse Library`
3. Add one of the following steps: `Chain Deployments`, `Consolidate Release Notes`, `Save Octopus Output Variable`
4. Return to Step Templates page and select one of the newly added templates, both return the below error.

```
Octopus v2018.7.1

TypeError: Cannot read property 'workerPoolId' of undefined

workerPoolId (/app/components/Actions/script/scriptAction.tsx:203:91)
Lf (/node_modules/react-dom/cjs/react-dom.production.min.js:147:337)
sf:a:{e=b.type;c=b.pendingProps;d=b.memoizedProps;if(nf())null===c&& (/node_modules/react-dom/cjs/react-dom.production.min.js:150:232)
e (/node_modules/react-dom/cjs/react-dom.production.min.js:182:349)
g (/node_modules/react-dom/cjs/react-dom.production.min.js:183:347)
p (/node_modules/react-dom/cjs/react-dom.production.min.js:184:366)
0;null!==a&&c;){c=!1;if(a.pendingWorkPriority===T||a.pendingWorkPriority>b)c=!0,a.pendingWorkPriority=b;null!==a.alternate&&(a.alternate.pendingWorkPriority===T||a.alternate.pendingWorkPriority>b)&&(c=!0,a.alternate.pendingWorkPriority=b);if(null===a["return"])if (/node_modules/react-dom/cjs/react-dom.production.min.js:188:389)
S (/node_modules/react-dom/cjs/react-dom.production.min.js:187:415)
enqueueSetState (/node_modules/react-dom/cjs/react-dom.production.min.js:140:181)
isMounted (/node_modules/react/cjs/react.production.min.js:12:343)
```

Sources:
https://help.octopus.com/t/error-when-viewing-community-step-templates/20821/3
https://help.octopus.com/t/save-output-variable-community-step/20832/2

**Comment by tw17** [comment]
duplicate of #4789 fixed in 2018.8.1

**Comment by queil** [comment]
@tw17 Still happening in  v2018.8.1

```
Octopus v2018.8.1

TypeError: Cannot read property 'workerPoolId' of undefined

workerPoolId (/app/components/Actions/script/scriptAction.tsx:347:114)
inlineScriptFormSections (/app/components/Actions/script/scriptAction.tsx:317:83)
Lf (/node_modules/react-dom/cjs/react-dom.production.min.js:147:337)
sf:a:{e=b.type;c=b.pendingProps;d=b.memoizedProps;if(nf())null===c&& (/node_modules/react-dom/cjs/react-dom.production.min.js:150:232)
e (/node_modules/react-dom/cjs/react-dom.production.min.js:182:349)
g (/node_modules/react-dom/cjs/react-dom.production.min.js:183:347)
p (/node_modules/react-dom/cjs/react-dom.production.min.js:184:366)
0;null!==a&&c;){c=!1;if(a.pendingWorkPriority===T||a.pendingWorkPriority>b)c=!0,a.pendingWorkPriority=b;null!==a.alternate&&(a.alternate.pendingWorkPriority===T||a.alternate.pendingWorkPriority>b)&&(c=!0,a.alternate.pendingWorkPriority=b);if(null===a["return"])if (/node_modules/react-dom/cjs/react-dom.production.min.js:188:389)
S (/node_modules/react-dom/cjs/react-dom.production.min.js:187:415)
enqueueSetState (/node_modules/react-dom/cjs/react-dom.production.min.js:140:181)
```
